src/vSphereHypervisorCollector.class.inc.php

```diff
@@ -93,6 +93,22 @@ public static function GetHypervisors()
 				Utils::Log(LOG_DEBUG, "Server {$oHypervisor->name}: {$oHypervisor->hardware->systemInfo->vendor} {$oHypervisor->hardware->systemInfo->model}");
 				Utils::Log(LOG_DEBUG, "Server software: {$oHypervisor->config->product->fullName} - API Version: {$oHypervisor->config->product->apiVersion}");
 
+				// Hypervisor System Serial Number, as reported by VSphere (Only tested on HPE servers)
+				$sSerialNumber='unknown';
```

Comment:
Please do not set `unknown` as serial number by default. To keep previous behaviour, it is best to keep empty.
```suggestion
				$sSerialNumber = '';
```